### DISCUSSION TITLE
2008/08/20/%E5%BE%AA%E7%8E%AF%E7%BB%93%E6%9E%84%E7%9A%84%E5%8F%8D%E9%A6%88%E8%8A%82%E7%82%B9

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#239] [Announcements]
# 2008/08/20/%E5%BE%AA%E7%8E%AF%E7%BB%93%E6%9E%84%E7%9A%84%E5%8F%8D%E9%A6%88%E8%8A%82%E7%82%B9

如果单纯是为了让下一次迭代使用上次迭代的数据，可以使用反馈节点，如下图所示。

https://blog.qizhen.xyz/2008/08/20/%E5%BE%AA%E7%8E%AF%E7%BB%93%E6%9E%84%E7%9A%84%E5%8F%8D%E9%A6%88%E8%8A%82%E7%82%B9

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Unknown
2008年11月4日 16:51 编辑
为什么在LV的ActiveX容器中插入MSCOMM32时总是提示错误
提示的错误为:类别未授权使用.在调用别的控件的时候有的可以. 

回复
杰
2008年09月3日 19:07 编辑
我从来没有用过这个反馈节点，呵呵。真的，虽然写LV好几年了。

回复
lisa
2008年08月25日 16:59 编辑
可以的话能帮忙将答案发到邮箱么？jianfeinv@126.com
谢谢

回复
lisa
2008年08月25日 16:57 编辑
想向高人请教个问题：Labview 中的显示控件“波形图”“波形图表”的区别是什么，谢谢

回复
